crates/holochain/src/conductor/conductor.rs

```diff
@@ -1463,13 +1463,16 @@ mod app_impls {
             };
 
             let (agent_key, derivation_details): (AgentPubKey, Option<DerivationDetailsInput>) =
-                if let Some(agent_key) = agent_key {
-                    if dpki.is_some() {
-                        // TODO: allow adding additional apps to an existing DPKI KeyRegistration.
-                        tracing::warn!("Using app with a pre-existing agent key: DPKI will not be used to manage keys for this app.");
-                    }
-                    (agent_key, None)
+            // pre-installed dpki for this app
+            if let Some(agent_key) = agent_key {
+                if dpki.is_some() {
+                    // TODO: allow adding additional apps to an existing DPKI KeyRegistration.
+                    tracing::warn!("Using app with a pre-existing agent key: DPKI will not be used to manage keys for this app.");
```

Comment:
```suggestion
                    tracing::warn!("App is being installed with an existing agent key: DPKI will not be used to manage keys for this app.");
```

The prefix seems redundant here